src/app/forms/input/input.html

```diff
@@ -0,0 +1,356 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+
+<h1>Layout - {{layout}} - {{grid ? 'grid' : 'no grid'}}</h1>
+
+<form class="clr-form clr-form-{{layout}}">
```

Comment:
The current forms have a title/label for the complete form. Not sure if we have that for the new forms or not but just wanted to note that here. cc: @reddolan 

![image](https://user-images.githubusercontent.com/1426805/37728186-007599ce-2d10-11e8-89ed-784a60e832e4.png)

![image](https://user-images.githubusercontent.com/1426805/37728231-1a1bc880-2d10-11e8-8729-88b357aa2630.png)

Reply:
Semantically, that label is just a piece of text like any header. I don't know why we'd need to do anything special to handle this case, just use header elements.